draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -369,25 +369,25 @@
    	<figure title="Basic CDNI Message Flow for Triggers" anchor="ure-basic-cdni-message-flow-for-triggers">
    <artwork><![CDATA[
    uCDN                                                   dCDN
-    |  (1) POST https://dcdn.example.com/triggers/uCDN      |
+    |  (1) POST https://dcdn.com/cit/uCDN                   |
```

Comment:
Per Slack conversation, noting that switching away from RFC2606 names is likely frowned upon by IETF, not to mention that dcdn.com belongs to an independent entity.